en/docs/develop/advanced-topics/manage-connectors.md

```diff
@@ -0,0 +1,58 @@
+# Manage Connectors 
+
+A connector in Choreo is a component you or another party have:
```

Comment:
Wondering whether we should avoid the term component here since it might cause confusion with what we refer to as component.